lib/src/Uncertainty/Algorithm/MetaModel/LinearModel/LinearModelResult.cxx

```diff
@@ -278,6 +316,10 @@ void LinearModelResult::load(Advocate & adv)
   adv.loadAttribute( "leverages_", leverages_ );
   adv.loadAttribute( "cookDistances_", cookDistances_ );
   adv.loadAttribute( "sigma2_", sigma2_ );
+  if (adv.hasAttribute("isModelSelection_"))
+    adv.loadAttribute("isModelSelection_", isModelSelection_);
+  else
```

Comment:
```suggestion
```

Reply:
This suggestion duplicates the next one.